.eslintrc.js

```diff
@@ -13,6 +19,8 @@ module.exports = {
     },
   ],
   rules: {
+    '@typescript-eslint/consistent-type-exports': 'error',
+    '@typescript-eslint/consistent-type-imports': 'error',
```

Comment:
```suggestion
    '@typescript-eslint/consistent-type-exports': [
      'error',
      {
        fixMixedExportsWithInlineTypeSpecifier: true,
      },
    ],
    '@typescript-eslint/consistent-type-imports': [
      'error',
      {
        fixStyle: 'inline-type-imports',
      },
    ],
```

Reply:
what do you think?

https://typescript-eslint.io/rules/consistent-type-imports/#fixstyle
https://typescript-eslint.io/rules/consistent-type-exports/#fixmixedexportswithinlinetypespecifier

Reply:
![image](https://github.com/fabricjs/fabric.js/assets/34343793/e29ffdc3-9a80-4ccd-b1b2-096ed0281b7a)

Reply:
aren't those catched with default values?

Reply:
even applying your config i don't get the extra changes.